libs/output-mapping/src/DeferredTasks/TableWriter/LoadTableTask.php

```diff
@@ -2,23 +2,12 @@
 
 namespace Keboola\OutputMapping\DeferredTasks\TableWriter;
 
-use Keboola\OutputMapping\Exception\OutputOperationException;
 use Keboola\StorageApi\Client;
-use Keboola\StorageApi\ClientException;
 
 class LoadTableTask extends AbstractLoadTableTask
 {
     public function startImport(Client $client)
     {
-        try {
-            $this->storageJobId = $client->queueTableImport($this->destination->getTableId(), $this->options);
-        } catch (ClientException $exception) {
-            if ($exception->getCode() === 403) {
-                throw new OutputOperationException(
```

Comment:
Tohle asi bylo blbe. Pokud to spravne chapu, tak se spis bere ze 
- `InvalidOutputException` je neco jako user error
- `OutputOperationException` app error

Reply:
Navic `OutputOperationException` nikde docker bundle nehandluje, jen `InvalidOutputException` https://github.com/keboola/docker-bundle/blob/1d2b9131fac6d93fed2c14048244f691a3a25fa6/src/Docker/Runner/DataLoader/DataLoader.php#L287

Reply:
Je to přesně jak píšeš, tohle je pozůstatatek z doby, kdy ještě nebyl UserExceptionInterface

Reply:
Jsem si prave nebyl jisty. Tak ja jdu zalozit issue, ze to nekdy musime upravit na novy standard

Reply:
Zalozil jsem na to issue https://keboola.atlassian.net/browse/PST-604